backend/fregepoc/repositories/constants.py

```diff
@@ -67,3 +68,4 @@ def get_languages_by_extension(
 register_extension("KOTLIN", [".kt"])
 register_extension("RUST", [".rs"])
 register_extension("GOLANG", [".go"])
+register_extension("TYPESCRIPT", [".ts", ".tsx"])
```

Comment:
I would rather be careful about adding `tsx` extension here. Are you sure the metrics used in analyzer would cope with understanding JSX?

Reply:
You are right, as it turns out lizard doesn't support .tsx files yet.